thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java

```diff
@@ -375,6 +398,43 @@ void updateDetectionPipeline(long detectionID, String payload, long startTime, l
     }
   }
 
+  private boolean isServiceAccount(ThirdEyePrincipal user) {
+    List<Predicate> predicates = new ArrayList<>();
+    predicates.add(Predicate.EQ("sessionKey", user.getSessionKey()));
+    predicates.add(Predicate.EQ("principalType", "SERVICE"));
+
+    List<SessionDTO> sessionDTO = this.sessionDAO.findByPredicate(Predicate.AND(predicates.toArray(new Predicate[0])));
+    return sessionDTO != null && !sessionDTO.isEmpty();
+  }
+
+  private void validateConfigOwner(ThirdEyePrincipal user, List<String> owners) {
+    if (owners == null || !owners.contains(user.getName())) {
```

Comment:
We need to update existing alerts created by service accounts.

Reply:
Good point. I could find 6 alerts, all created by career-comms service account. I will update them during deployment.